frontend/packages/ceph-storage-plugin/integration-tests/tests/add-capacity.scenario.ts

```diff
@@ -12,11 +12,8 @@ import {
   storageClusterRow,
 } from '../views/add-capacity.view';
 
-const namespace = 'openshift-storage';
-const kind = 'storagecluster';
-const storageCluster = JSON.parse(
-  execSync(`kubectl get -o json -n ${namespace} ${kind}`).toString(),
-);
+const NAMESPACE = 'openshift-storage';
+const KIND = 'storagecluster';
```

Comment:
put these  in consts folder

Reply:
Its already done. @afreen23 Please rebase with my PR.